config/secrets.yml

```diff
@@ -134,7 +134,7 @@ test:
   uphold_client_id: test_client_id
   uphold_client_secret: test_client_secret
   uphold_scope: cards:read,cards:write,user:read
-  uphold_api_uri: "https://uphold-api.example.com"
+  uphold_api_uri: "https://api-sandbox.uphold.com"
```

Comment:
I don't think we should be making any external requests for tests.

If you replace the failing travis test with this it should work:
```ruby
  test "when online returns true when the wallet can be set" do
    prev_offline = Rails.application.secrets[:api_eyeshade_offline]
    begin
      Rails.application.secrets[:api_eyeshade_offline] = false

      publisher = publishers(:verified)
      publisher.uphold_access_parameters = "{\"foo\":\"bar\"}"
      wallet = "{\"wallet\":\"abc123\"}"

      stub_request(:put, /v1\/owners\/#{URI.escape(publisher.owner_identifier)}\/wallet/).
          with(body: "{\n  \"provider\": \"uphold\",\n  \"parameters\": {\"foo\":\"bar\",\"server\":\"https://uphold-api.example.com\"}\n}\n").
          to_return(status: 200, body: wallet, headers: {})

      result = PublisherWalletSetter.new(publisher: publisher).perform
      assert_equal 200, result.status
    ensure
      Rails.application.secrets[:api_eyeshade_offline] = prev_offline
    end
  end
```

The problem was the the stub request did not match the request made, and thus returned `<Faraday::ConnectionFailed>` and `nil` as the result